static/bidder-info/aax.yaml

```diff
@@ -0,0 +1,19 @@
+maintainer:
+  email: product@aax.media
+gvlVendorID: 720
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
+    url: https://c.aaxads.com/aacxc.php?fv=1&wbsh=psa&ryvlg=setstatuscode&bidder=aax&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    userMacro: ${UUID}
```

Comment:
When I test this user sync url, I get a 200 response instead of a redirect.

Reply:
Isn't that the case with every bidder as well ?

Reply:
@SyntaxNode Can you post any ref. of bidder adapter .yaml where we can see this behaviour ?

Reply:
@product-aax Certainly. You can look at `appnexus.yaml` and `visx.yaml` for fully compliant examples. For `appnexus.yaml`:

This is what they define in as their user sync:
```
redirect:
  url: "https://ib.adnxs.com/getuid?{{.RedirectURL}}"
  userMacro: "$UID"
```

Here is an example of the generated url by the cookie sync endpoint. This example comes from the AppNexus Prebid Server host.
```
https://ib.adnxs.com/getuid?https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dadnxs%26gdpr%3D0%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID
```

The redirect url looks like the following when it is url decoded:
```
https://prebid.adnxs.com/pbs/v1/setuid?bidder=adnxs&gdpr=0&gdpr_consent=&f=i&uid=$UID
```

The AppNexus user sync endpoint will decode the redirect url, replace the `$UID` macro with a the user's value from the AppNexus cookie and respond with the following redirect:
```
https://ib.adnxs.com/prebid/setuid?bidder=visx&gdpr=0&gdpr_consent=&uid=c8b707fe-bb40-4862-a12c-c0603bfaeb0f
```

It is common for adapters to use an allow list for the redirect domain name to protect against abuse, although that's not a requirement we have for Prebid Server.

Reply:
ok @SyntaxNode, I got it. So see, at out end we have domain whitelisting, so that was the reason when you tried hitting the endpoint you got 200 and not 302 because that redirect url domain was not in our whitelist. I tried hitting the endpoint with rubicon redirect url and it worked. You can now test it with appnexus as well since we have added it in our  list .
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/103228925/164709739-c03dcdc1-5f7e-4513-a16d-aaf4acc304ff.png">

Reply:
@SyntaxNode, We have removed the whitelisting service from our end. I think we are good to go. Thanks.

Reply:
I tried to test user sync URL and it returns just http 200, no redirects. On the screenshot you provided earlier I see the redirect URL is not encoded, it's uses as is. @SyntaxNode is this a valid case?
My test URL looks like this:
```
https://c.aaxads.com/aacxc.php?fv=1&wbsh=psa&ryvlg=setstatuscode&bidder=aax&gdpr=&gdpr_consent=&us_privacy=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Daax%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24%7BUUID%7D
```
Other than this code looks good to me.

Reply:
Good point @VeronikaSolovei9. The redirect url is always url encoded for safety. @product-aax can you confirm the redirect works correctly url encoded?